rearrange/environment.py

```diff
@@ -181,16 +185,13 @@ def __init__(
             when initializing the `ai2thor.controller.Controller` (e.g. width/height).
         """
         if ai2thor.__version__ is not None:  # Allows for custom THOR installs
-            for v in REQUIRED_THOR_VERSION.split(","):
-                operators = ["<=", ">=", "==", "<", ">"]
-                if any(op in v for op in operators):
-                    op = next(op for op in operators if op in v)
-                    v = f"{op}'{v.replace(op, '')}'"
-                    correct_version = eval(f"'{ai2thor.__version__}'{v}")
-                else:
-                    correct_version = ai2thor.__version__ == v
-                if not correct_version:
-                    raise ValueError(f"Please use AI2-THOR v{REQUIRED_THOR_VERSION}")
+            if ai2thor.__version__ not in ["0.0.1", None] and version.parse(
```

Comment:
Why allow version `0.0.1`?

Reply:
"0.0.1" is the version that AI2-THOR reports when using a custom build. If someone is using a custom build then they can do what they want.